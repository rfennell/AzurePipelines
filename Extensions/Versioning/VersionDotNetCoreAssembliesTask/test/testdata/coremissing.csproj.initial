<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.1</TargetFramework>
    <AssemblyName>VersionedLibrary</AssemblyName>
    <Description>A description</Description>
  </PropertyGroup>

</Project>
