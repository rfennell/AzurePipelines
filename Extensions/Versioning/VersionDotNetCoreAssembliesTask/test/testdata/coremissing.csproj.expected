<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.1</TargetFramework><AssemblyVersion>9.9.9.9</AssemblyVersion>
    <AssemblyName>VersionedLibrary</AssemblyName>
    <Description>A description</Description>
  </PropertyGroup>

</Project>
